src/module.c

```diff
@@ -292,6 +320,12 @@ int TSDB_info(RedisModuleCtx *ctx, RedisModuleString **argv, int argc) {
     RedisModule_ReplyWithLongLong(ctx, RedisModule_DictSize(series->chunks));
     RedisModule_ReplyWithSimpleString(ctx, "chunkSize");
     RedisModule_ReplyWithLongLong(ctx, series->chunkSizeBytes);
+    RedisModule_ReplyWithSimpleString(ctx, "duplicatePolicy");
+    if (series->duplicatePolicy != DP_NONE) {
```

Comment:
@ashtul if a policy was not set on a key its considered as "none", one can provide a policy on a key level if it didn't provide any its considered as if the key doesn't have any specific policy and will use the database wide policy (default is block).
Check the `Precedence order` in `configuration.md`

Reply:
Instead of being 'none' and then checked against global every time, it can be set as the global default. 
From that point on, only the series can be set.